paper/myst.yml

```
@@ -1,49 +1,50 @@
 # See docs at: https://mystmd.org/guide/frontmatter
 version: 1
 project:
-  # id: ubcgif-YEAR-AUTHORS-JOURNAL
-  # title: TITLE OF PAPER
-  # description: SHORT DESCRIPTION HERE
+  id: ubcgif-2019-heagy-oldenburg-gji
```

```suggestion
  id: ubcgif-2019-heagy-oldenburg-computers-geosciences
```

This is a different journal